man/smc2_parameters.Rd

```diff
@@ -13,18 +13,18 @@ used.
 \section{Methods}{
 \subsection{Public methods}{
 \itemize{
-\item \href{#method-new}{\code{smc2_parameters$new()}}
-\item \href{#method-sample}{\code{smc2_parameters$sample()}}
-\item \href{#method-names}{\code{smc2_parameters$names()}}
-\item \href{#method-summary}{\code{smc2_parameters$summary()}}
-\item \href{#method-prior}{\code{smc2_parameters$prior()}}
-\item \href{#method-propose}{\code{smc2_parameters$propose()}}
-\item \href{#method-model}{\code{smc2_parameters$model()}}
+\item \href{#method-smc2_parameters-new}{\code{smc2_parameters$new()}}
```

Comment:
looks like when I generated docs they came out in a different format, do you know why?

Reply:
You need to upgrade roxygen2 I think

```
> packageVersion("roxygen2")
[1] ‘7.2.0’
```

I think the last version update has a lot of churny change